### PR TITLE
Creating packer to build the amis and ansible to provisioning the users in the bastion host

### PR DESCRIPTION
**Pull request by masterplanx**

### CODE REVIEW

File: terraform/bastion_instance.tf [hunk 1]
```diff
@@ -0,0 +1,41 @@
+resource "aws_instance" "bastion_instance_flugel_poc" {
+  ami           = "${lookup(var.AMIS, var.AWS_REGION)}"
```
Comment (diegows) [inline terraform/bastion_instance.tf]: I see you are using data sources to get the latest image generated by Packer, but I do no see you are using the data source here to use that image. You should be using data source here.

File: terraform/bastion_instance.tf [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  provisioner "local-exec" {
+    environment {
+        PUBLIC_IP  = "${self.public_ip}"
+        PRIVATE_IP = "${self.private_ip}"
+    }
+
+    working_dir = "../ansible/create_users/"
+    command     = "ansible-playbook --private-key ../${var.PATH_TO_PRIVATE_KEY} playbook.yml -i ${self.public_ip},"
```
Comment (diegows) [inline terraform/bastion_instance.tf]: Do not provision instance here, provision everything on packer.

**Review by diegows — CHANGES_REQUESTED** [review]

File: terraform/bastion_instance.tf (hunk 2 above)
Comment (masterplanx) [reply terraform/bastion_instance.tf]: It's done, I created the ansible provisioner in packer to bastion and private instances, and I removed provisioner from terraform.

File: terraform/bastion_instance.tf (hunk 1 above)
Comment (masterplanx) [reply terraform/bastion_instance.tf]: Added datasource in the bastion tf

**Review by diegows — APPROVED** [review]

**Pull request merged**